src/messages.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="messages.css">
+    <title>Messages</title>
+</head>
+<body>
+    <div class="container">
+        <form action="/https://jsonplaceholder.typicode.com/posts" method="POST">
```

Comment:
URLs dont start with slash :)

```suggestion
        <form action="https://jsonplaceholder.typicode.com/posts" method="POST">
```